portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -45,10 +62,20 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
       //Get the input from the form
       String text = getParameter(request, "text-input", "This is an empty comment!");
```

Comment:
Why do you need "This is an empty comment!" line?
You seems do not need getParameter method because you can directly call  request.getParameter and then check for null value. Create extra methods only when  they bring some value to the code.

Reply:
Ok, sounds good! Sorry about that!